packages/@aws-cdk/cx-api/lib/versioning.ts

```diff
@@ -27,14 +42,29 @@ export function verifyManifestVersion(manifetVersion: string) {
 
   // if framework > cli, we require a newer cli version
   if (semver.gt(frameworkVersion, toolkitVersion)) {
-    throw new Error(`CDK CLI >= ${frameworkVersion} is required to interact with this app`);
+    throw new Error(`A newer version of the CDK CLI (>= ${frameworkVersion}) is necessary to interact with this app`);
```

Comment:
For future generations maybe rename “frameworkVersion” to “frameworkAssemblyVersion” or “appAssemblyVersion”